packages/core/test/interpreter/takeSync.js

```diff
@@ -443,57 +443,7 @@ test('inter-saga send/aknowledge handling (via buffered channel)', assert => {
     })
 })
 
-test('inter-saga fork/take back from forked child', assert => {
```

Comment:
could u point me to the duplicate of this one?

Reply:
[L446-L494](https://github.com/redux-saga/redux-saga/blob/1b4ba30c063bcb28ba4ff7ed5f430fe88d99dec3/packages/core/test/interpreter/takeSync.js#L446-L494)  and [L496-L544](https://github.com/redux-saga/redux-saga/blob/1b4ba30c063bcb28ba4ff7ed5f430fe88d99dec3/packages/core/test/interpreter/takeSync.js#L496-L544)

Reply:
Probably some wrong merge artifact 😅